ansible-fmw/templates/WDT_SSO_template-baseline.yaml.j2

```diff
@@ -0,0 +1,271 @@
+topology:
+    Name: '{{ DOMAIN_NAME }}'
+    DomainVersion: 12.2.1.3.0
+    ProductionModeEnabled: true
+    NMProperties:
+        JavaHome: '{{ JAVA_HOME }}'
+        PropertiesVersion: 12.2.1.3.0
+        weblogic.StartScriptName: startWebLogic.sh
+    Server:
+        AdminServer:
+            Machine: oam_Machine_1
+            WebServer:
+                WebServerLog:
+                    LogFileFormat: extended
+                    ELFFields: date time cs-method ctx-ecid ctx-rid cs-uri sc-status
+                        bytes
+        oam_server1:
+            ListenPort: 14100
+            Machine: oam_Machine_1
+            WebServer:
+                WebServerLog:
+                    LogFileFormat: extended
+                    ELFFields: date time cs-method ctx-ecid ctx-rid cs-uri sc-status
+                        bytes
+        oam_policy_mgr1:
+            ListenPort: 14150
+            Machine: oam_Machine_1
+            WebServer:
+                WebServerLog:
+                    LogFileFormat: extended
+                    ELFFields: date time cs-method ctx-ecid ctx-rid cs-uri sc-status
+                        bytes
+    Machine:
+        oam_Machine_1: {}
+    
+resources:
+    StartupClass:
+        WSM Startup Class:
+            ClassName: oracle.wsm.config.WSMServerStartupShutdownProvider
+            Target: oam_policy_mgr1,AdminServer,oam_server1
+        Web Services Startup Class:
+            LoadBeforeAppActivation: true
+            DeploymentOrder: 150
+            LoadBeforeAppDeployments: true
+            ClassName: oracle.j2ee.ws.server.WebServiceServerStartup
+            Target: oam_server1,oam_policy_mgr1,AdminServer
+        JRF Startup Class:
+            LoadBeforeAppActivation: true
+            DeploymentOrder: 110
+            LoadBeforeAppDeployments: true
+            ClassName: oracle.jrf.wls.JRFStartup
+            Target: oam_server1,oam_policy_mgr1,AdminServer
+        ODL-Startup:
+            LoadBeforeAppActivation: true
+            DeploymentOrder: 145
+            LoadBeforeAppDeployments: true
+            ClassName: oracle.core.ojdl.weblogic.ODLConfiguration
+            Target: oam_server1,AdminServer,oam_policy_mgr1
+        DMS-Startup:
+            LoadBeforeAppActivation: true
+            DeploymentOrder: 150
+            LoadBeforeAppDeployments: true
+            ClassName: oracle.dms.wls.DMSStartup
+            Target: oam_server1,oam_policy_mgr1,AdminServer
+        AWT Application Context Startup Class:
+            LoadBeforeAppActivation: true
+            DeploymentOrder: 150
+            LoadBeforeAppDeployments: true
+            ClassName: oracle.jrf.AppContextStartup
+            Target: oam_policy_mgr1,AdminServer,oam_server1
+    ShutdownClass:
+        DMSShutdown:
+            DeploymentOrder: 150
+            ClassName: oracle.dms.wls.DMSShutdown
+            Target: AdminServer,oam_policy_mgr1,oam_server1
+    JDBCSystemResource:
```

Comment:
we intentionally avoided using any DB resources, because this will require setting up DB as well.  
introducing this will lead to unwanted complexity